### ISSUE TITLE
Error when importing vue-simple-calendar/dist/style.css

### ISSUE THREAD

**Issue by Cayaf89** [richardtallent/vue-simple-calendar#221]
Hello !

When I'm importing the style.css in my component, I get this error :

`Module not found: Error: Package path ./dist/style.css is not exported from package /node_modules/vue-simple-calendar (see exports field in /node_modules/vue-simple-calendar/package.json)
ERROR in ./src/views/TabCalendarPage.vue?vue&type=script&lang=js (./node_modules/babel-loader/lib/index.js??clonedRuleSet-40.use[0]!./node_modules/vue-loader/dist/index.js??ruleSet[0].use[0]!./src/views/TabCalendarPage.vue?vue&type=script&lang=js) 4:0-44
[vue-cli-service] Module not found: Error: Package path ./dist/style.css is not exported from package /crew-party/node_modules/vue-simple-calendar (see exports field in /node_modules/vue-simple-calendar/package.json)`

Can you add this to your package.json please ?

`"./dist/style.css": {
        "import": "./dist/style.css",
        "require": "./dist/style.css"
},`

I made a pull request for this : #222 

Thank you !

**Comment by ulcuber** [comment]
At least `package.json` has `style`
https://stackoverflow.com/questions/32037150/style-field-in-package-json

But workaround in scss:
```scss
@import 'vue-simple-calendar';
@import '~vue-simple-calendar';
@import '~vue-simple-calendar/dist/style.css';
```
may not work too

**Comment by vzhikness** [comment]
I got a similar issue with importing `style.css` in my Vue 3 app with Vite. I was importing it like that:

```js
<style lang="postcss">
@import 'vue-simple-calendar/dist/style.css';
</style>
```

And the error was:
```
[vite:css] [postcss] Missing "./dist/style.css" export in "vue-simple-calendar" package
file: .../event-calendar.vue?vue&type=style&index=0&lang.postcss:undefined:undefined
error during build:
Error: [postcss] Missing "./dist/style.css" export in "vue-simple-calendar" package
```

Then I replaced import with this:

```js
<style lang="postcss">
@import 'vue-simple-calendar';
</style>
```

And it's working. @ulcuber thanks for the idea!

**Comment by richardtallent** [comment]
Fixed in 7.0, thanks!
